README.rst

```diff
@@ -137,13 +141,13 @@ To build the PHP driver for Windows:
    For :code:`<arch>`, :code:`<build>`, :code:`<visual studio version>`, and :code:`<path to PHP SDK>`, specify the same values
    that you used in the previous step.
 
-   For :code:`<full PHP version>`, specify the full version number of PHP that you want to install (e.g. :code:`7.2.24`).
+   For :code:`<full PHP version>`, specify the full version number of PHP that you want to install (e.g. :code:`8.1.18`).
```

Comment:
Change from
"the full version number of PHP that you want to install"
to something like
"the full version number of the PHP binary you installed"

Reply:
Done